README.md

```diff
@@ -1,126 +1,195 @@
+
+---
 
-> **Note**<br/>
-> For additional automation capabilities on the Robocorp platform, please refer to the wide array of libraries and keywords available in [RPA Framework](https://robocorp.com/docs/libraries/rpa-framework).
+## Why use Robocorp AI Actions
 
-## CLI
+- ❤️ “when it comes to automation, the Robocorp suite is the best one” _[/u/disturbing_nickname](https://old.reddit.com/r/rpa/comments/18qqspn/codeonly_rpa_pet_project/kez2jds/?context=3)_
+- ❤️ “Robocorp seems to be a good player in this domain” _[/u/thankred](https://old.reddit.com/r/rpa/comments/18r5gne/recommendation_for_open_source_or_somewhat_less/kez6aw6/?context=3)_
+- ❤️ “Since you know Python, check out Robocorp. Their product is crazy good.” _[/u/Uomis](https://old.reddit.com/r/rpa/comments/18n5sah/c/ke8qz2g?context=3)_
 
-> **Warning**<br/>
-> Robo CLI is not yet recommended for production use, while libraries are ready for prime time. Refer to Robocorp's production tooling for managing Python projects [here](https://robocorp.com/docs/quickstart-guide/python).
+Robocorp stack is hands down the easiest way to give AI agents more capabilities. It’s an end-to-end stack supporting every type of connection between AI and your apps and data. You are in control where to run the code and everything is built for easiness, security, and scalability.
 
-The `robo` CLI is used to manage and run your projects during development, and it bundles your project for deployment.
+- 🔐 **Decouple AI and Actions that touches your data/apps** - Clarity and security with segregation of duties between your AI agent and code that touches your data and apps. Build `@action` and use from multiple AI frameworks.
+- 🏎️ **Develop Actions faster with `robocorp` automation libraries** - Robocorp libraries and the Python ecosystem let's you act on anything - from data to API to Browser to Desktops.
+- 🕵️ **Observability out of the box** - Log and trace every `@action` run automatically without a single `print` statement. 
+- 🤯 **No-pain Python environment management** - Don't do [this](https://xkcd.com/1987/). Robocorp manages a full Python environment for your actions with ease.
+- 🚀 **Deploy with zero config and infra** - One step deployment, and you'll be connecting your `@action` to AI apps like Langchain and OpenAI GPTs in seconds.
 
-#### `robo new`
+## Inspo
 
-Create an entirely new Python project, isolated from the rest of your system. No need to install Python separately or manually manage virtual environments.
+Check out these example projects for inspiration.
 
-#### `robo run`
+- 🤡 Get a random joke or jokes per theme. Showcases how easy it is to work with APIs. (Coming soon, credit XX for the idea)
+- 🕸️ Open a local Playwright browser and make some Google searches. (Coming soon)
+- 🖥️ Fetch contents of `.txt` and `.pdf` files from your local machine's folder in real time. (Coming soon)
```

Comment:
formulate a little bit differently to promote secure, strictly controlled access to the local machine resources (explicitly configured folder etc) ?